### PR TITLE
Enable stargate feature on contracts

### PR DESCRIPTION
**Pull request by CyberHoward**
This enables the stargate feature on all contracts, allowing users to execute stargate messages on their accounts and over IBC

### CODE REVIEW

**Review by Buckram123 — COMMENTED** [review]
New wasms is the same, stargate was already enabled for all contracts it seems

File: framework/Cargo.toml [hunk 1]
```diff
@@ -32,7 +32,7 @@ license = "GPL-3.0-or-later"
 keywords = ["cosmos", "cosmwasm", "framework"]
 
 [workspace.dependencies]
-cosmwasm-std = { version = "1.5.0", features = ["cosmwasm_1_2", "stargate"] }
```
Comment (CyberHoward) [inline framework/Cargo.toml]: even if the stargate feature is enabled implicitly I'd still prefer having it enabled explicitly, no?

File: interchain/interchain-tests/src/interchain_accounts.rs [hunk 2]
```diff
@@ -161,6 +161,79 @@ mod test {
...
+            "ics20-1",
+            None, // Unordered channel
+        )?;
+
+        // The user on origin chain wants to change the account description
+        let ibc_transfer_result = origin_account.manager.execute_on_module(
+            PROXY,
+            &proxy::ExecuteMsg::ModuleAction {
+                msgs: vec![cosmwasm_std::CosmosMsg::Ibc(
+                    cosmwasm_std::IbcMsg::Transfer {
```
Comment (CyberHoward) [inline interchain/interchain-tests/src/interchain_accounts.rs:201]: This isn't a stargate message afaik.

Comment (Buckram123) [reply interchain/interchain-tests/src/interchain_accounts.rs:201]: It's not stargate message, but message enabled by stargate feature, what are we supposed to be checking here?

File: framework/Cargo.toml (hunk 1 above)
Comment (Buckram123) [reply framework/Cargo.toml]: It's enabled on proxy explicitly

File: interchain/interchain-tests/src/interchain_accounts.rs (hunk 2 above)
Comment (CyberHoward) [reply interchain/interchain-tests/src/interchain_accounts.rs:201]: Ok, just looked it up, you're right. Would have expected it to be under the ibc feature. 
LGTM!

File: framework/Cargo.toml (hunk 1 above)
Comment (CyberHoward) [reply framework/Cargo.toml]: and ibc-client / host / manager?

**Pull request merged**